src/main/groovy/edu/oregonstate/mist/locations/mapper/LocationMapper.groovy

```diff
@@ -26,13 +26,17 @@ class LocationMapper  {
             geoLocation: createGeoLocation(campusMapLocation.latitude,
                                             campusMapLocation.longitude),
             geometry: new Geometry(
-                coordinates: campusMapLocation.coordinates),
+                coordinates: campusMapLocation.coordinates,
+                type: campusMapLocation.coordinatesType),
             address: campusMapLocation.address,
             summary: campusMapLocation.shortDescription,
             description: campusMapLocation.description,
             thumbnails: [getImageUrl(campusMapLocation.thumbnail)] - null,
             images: [getImageUrl(campusMapLocation.largerImage)] - null,
-            type: Constants.TYPE_BUILDING
+            type: Constants.TYPE_BUILDING,
```

Comment:
ArcGis is merged into a campus map location object in this method: https://github.com/osu-mist/locations-api/blob/feature/CO-815-gender-inclusive-rr-implementation/src/main/groovy/edu/oregonstate/mist/locations/db/LocationDAO.groovy#L97-L123